packages/ozone/src/config.ts

```diff
@@ -0,0 +1,157 @@
+import assert from 'assert'
+
+export interface ServerConfigValues {
+  version: string
+  debugMode?: boolean
+  port?: number
+  publicUrl?: string
+  serverDid: string
+  appviewUrl: string
+  appviewDid?: string
+  pdsUrl?: string
+  pdsDid?: string
+  dbPostgresUrl: string
+  dbPostgresSchema?: string
+  didPlcUrl: string
+  labelerDid: string
+  adminPassword: string
+  moderatorPassword?: string
+  triagePassword?: string
+}
+
+export class ServerConfig {
+  private assignedPort?: number
+  constructor(private cfg: ServerConfigValues) {}
+
+  static readEnv(overrides?: Partial<ServerConfigValues>) {
+    const version = process.env.BSKY_VERSION || '0.0.0'
+    const debugMode = process.env.NODE_ENV !== 'production'
+    const publicUrl = process.env.PUBLIC_URL || undefined
+    const serverDid = process.env.SERVER_DID || 'did:example:test'
+    const envPort = parseInt(process.env.PORT || '', 10)
+    const port = isNaN(envPort) ? 2584 : envPort
+    const appviewUrl = process.env.APPVIEW_URL
+    assert(appviewUrl)
+    const appviewDid = process.env.APPVIEW_DID
+    const pdsUrl = process.env.PDS_URL
+    const pdsDid = process.env.PDS_DID
+    const dbPostgresUrl =
+      overrides?.dbPostgresUrl || process.env.DB_POSTGRES_URL
+    assert(dbPostgresUrl)
+    const dbPostgresSchema = process.env.DB_POSTGRES_SCHEMA
+    const didPlcUrl = process.env.DID_PLC_URL || 'http://localhost:2582'
+    const adminPassword = process.env.ADMIN_PASSWORD || 'admin'
+    const moderatorPassword = process.env.MODERATOR_PASSWORD || undefined
+    const triagePassword = process.env.TRIAGE_PASSWORD || undefined
+    const labelerDid = process.env.LABELER_DID || 'did:example:labeler'
```

Comment:
I wonder if we should use the convention of prefixing the env vars with the service name `OZONE_` as we've done on the pds (and recently on the appview).

Reply:
yup agreed - i'll edit all of those 👍

Reply:
ended up refactoring all the config to look more similar to the PDS